src/components/v2/ApyChart/index.tsx

```diff
@@ -0,0 +1,152 @@
+            tickMargin={styles.tickMargin}
+            style={styles.axis}
+          />
+          <YAxis
+            axisLine={false}
+            tickLine={false}
+            tickFormatter={formatToReadablePercentage}
+            tickMargin={styles.tickMargin}
+            stroke={styles.accessoryColor}
+            style={styles.axis}
+            domain={[0, 'dataMax + 10']}
+          />
+          <Tooltip
+            isAnimationActive={false}
+            cursor={styles.cursor}
+            content={({ payload }) =>
+              payload && payload[0] ? (
+                <div css={styles.tooltipContainer}>
+                  <div css={styles.tooltipItem}>
+                    <Trans
+                      i18nKey={
+                        // Translation keys: do not remove this comment
+                        // t('apyChart.tooltipItems.supplyApy')
+                        // t('apyChart.tooltipItems.borrowApy')
+                        type === 'supply'
+                          ? 'apyChart.tooltipItems.supplyApy'
+                          : 'apyChart.tooltipItems.borrowApy'
+                      }
+                      components={{
+                        Label: <Typography css={styles.tooltipItemLabel} variant="tiny" />,
+                        Value: <Typography css={styles.tooltipItemValue} variant="small1" />,
+                      }}
+                      values={{
+                        percentage: formatToReadablePercentage((payload[0].payload as IItem).apy),
+                      }}
+                    />
+                  </div>
+
+                  <div css={styles.tooltipItem}>
+                    <Trans
+                      i18nKey={
+                        // Translation keys: do not remove this comment
+                        // t('apyChart.tooltipItems.totalSupply')
+                        // t('apyChart.tooltipItems.totalBorrow')
+                        type === 'supply'
+                          ? 'apyChart.tooltipItems.totalSupply'
+                          : 'apyChart.tooltipItems.totalBorrow'
+                      }
+                      components={{
+                        Label: <Typography css={styles.tooltipItemLabel} variant="tiny" />,
+                        Value: <Typography css={styles.tooltipItemValue} variant="small1" />,
+                      }}
+                      values={{
+                        balance: formatCentsToReadableValue({
+                          value: (payload[0].payload as IItem).balanceCents,
+                          shorthand: true,
+                        }),
+                      }}
+                    />
+                  </div>
+                </div>
+              ) : (
+                // Although the type definition of "content" allows an undefined
+                // value to be returned, it actually creates a runtime error if
+                // we do so. For that reason we return a fragment instead
+                <></>
```

Comment:
null should be returned in place of rendering `undefined`. We can also remove the comment if we return null

Reply:
That's much nicer 👍